app/src/main/java/ar/com/wolox/android/example/ui/home/HomeFragment.kt

```diff
@@ -0,0 +1,78 @@
+package ar.com.wolox.android.example.ui.home
+
+import androidx.core.util.Pair
+import androidx.fragment.app.Fragment
+import ar.com.wolox.android.example.ui.news.NewsFragment
+import ar.com.wolox.android.example.ui.profile.ProfileFragment
+import ar.com.wolox.wolmo.core.adapter.viewpager.SimpleFragmentPagerAdapter
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import com.google.android.material.tabs.TabLayout
+import kotlinx.android.synthetic.main.fragment_home.*
+import kotlinx.android.synthetic.main.fragment_viewpager.vViewPager
+import javax.inject.Inject
+import android.os.Build
+import android.view.WindowManager
+import androidx.core.content.ContextCompat
+import ar.com.wolox.android.R
+
+class HomeFragment @javax.inject.Inject constructor() : WolmoFragment<BasePresenter<Any>>() {
+
+    @Inject
+    internal lateinit var newsFragment: NewsFragment
+    @Inject
+    internal lateinit var profileFragment: ProfileFragment
+    private lateinit var fragmentPagerAdapter: SimpleFragmentPagerAdapter
+
+    private val newsTab = 0
+    private val profileTab = 1
+
+    override fun layout(): Int = R.layout.fragment_home
+
+    override fun init() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+            activity!!.window.addFlags(WindowManager.LayoutParams.FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS)
```

Comment:
Replace all `activity!!` with requireActivity(). Moreover, one pro that kotlin gives us, is the optimization of code lines/length. You can do `requireActivity().window.apply{}`. Read a little of `apply{}.`